Create README for Web Components organization (#1)
FACT-Finder-Web-Components/.github: profile/README.md

> # FACT Finder Web Components 🚀

Reviewer: ```suggestion
# FactFinder Web Components 🚀
```

### Integrating FactFinder has never been easier

> As of now, FactFinder can be integrated into your online shop as easy as a YouTube video or a Google map – by means of prefabricated HTML elements, so called Web Components.

Reviewer: I think "as easy as a YT video" is not quite accurate.

```suggestion
FactFinder can be integrated easily into your online shop by means of prefabricated HTML elements, so called Web Components.
These Web Components offer fully functional features such as searching, suggestions, filtering, sorting, and many more.
```

We are constantly updating the SDKs so that you can use the latest versions of the above-mentioned e-commerce platforms.

#### Magento2 Module

- [SDK v4.x](https://github.com/FACT-Finder-Web-Components/magento2-module/tree/release/4.x) &rarr; Web Components v4.x, Magento v2.4.4 or higher
- [SDK v5.x](https://github.com/FACT-Finder-Web-Components/magento2-module/tree/release/5.x) &rarr; Web Components v5.x, Magento v2.4.4 or higher

#### Shopware6 Plugin

- [SDK v4.x](https://github.com/FACT-Finder-Web-Components/shopware6-plugin/tree/release/4.x) &rarr; Web Components v4.x, Shopware v6.4
- [SDK v5.x](https://github.com/FACT-Finder-Web-Components/shopware6-plugin/tree/release/5.x) &rarr; Web Components v4.x, Shopware v6.5
- [SDK v6.x](https://github.com/FACT-Finder-Web-Components/shopware6-plugin/tree/release/6.x) &rarr; Web Components v5.x, Shopware v6.6

#### Oxid Module

- [SDK v4.x](https://github.com/FACT-Finder-Web-Components/oxid-eshop-module) &rarr; Web Components v4.x, Oxid v6
- [SDK v5.x](https://github.com/FACT-Finder-Web-Components/oxid-eshop-module/tree/release/5.x) &rarr; Web Components v4.x, Oxid v7

> - SDK v6.x &rarr; Web Components v5.x, Oxid v7 (comming soon, work in progress)

Reviewer: ```suggestion
- SDK v6.x &rarr; Web Components v5.x, Oxid v7 (coming soon, work in progress)
```
